backend/codesirius/problems/serializers/tag.py

```
@@ -1,61 +1,10 @@
-                "description", instance.description
-            )
-            instance.save()
-            logger.info(f"Tag with ID: {instance.id} updated successfully")
-            return instance
-        except Exception as e:
-            logger.error(f"Error updating tag with ID: {instance.id}: {e}")
-            raise APIException("Error while updating tag")
+class TagSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Tag
+        fields = ["id", "name", "description"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)